scripts/test_env.py

```diff
@@ -140,24 +140,31 @@ def pos_int(arg):
         raise argparse.ArgumentTypeError(f"Argument {arg} must be a positive integer")
 
 def copy_or_prompt_build_bin(bin_name: str, root_dir: pathlib.Path, target_dir: pathlib.Path):
-    try:
-        shutil.copy2(f"{root_dir}/target/debug/{bin_name}", target_dir)
-    except FileNotFoundError:
-        logging.error(f"The binary `{bin_name}` wasn't found in `{root_dir}` directory")
-        while True:
-            prompt = input(f"Build it by running `cargo build --bin {bin_name}`? (Y/n)\n")
-            if prompt.lower() in ["y", "yes", ""]:
-                subprocess.run(
-                    ["cargo", "build", "--bin", bin_name],
-                    cwd=root_dir
-                )
-                shutil.copy2(f"{root_dir}/target/debug/{bin_name}", target_dir)
-                break
-            elif prompt.lower() in ["n", "no"]:
-                logging.critical("Can't launch the network without the binary. Aborting...")
-                sys.exit(4)
-            else:
-                logging.error("Please answer with either `y[es]` or `n[o])")
+    binary_paths = [
+        f"{root_dir}/target/debug/{bin_name}",
+        f"{root_dir}/target/release/{bin_name}"
```

Comment:
I don't think this is a particularly good way to do it. This script should take another parameter to decide whether to use `debug` build or `release` build. It shouldn't be implicit like this

Reply:
Got you, but this edit doesn't seem relevant anymore since `debug` mode is faster for test environment purposes. If we will need it I think it will be better if I do it in separate issue.